include/avif/internal.h

```diff
@@ -137,8 +137,8 @@ typedef struct avifReformatState
     avifPixelFormatInfo formatInfo;
 
     // LUTs for going from YUV limited/full unorm -> full range RGB FP32
-    float unormFloatTableY[1 << 12];
-    float unormFloatTableUV[1 << 12];
+    float unormFloatTableY[1 << 16];
+    float unormFloatTableUV[1 << 16];
```

Comment:
Note that we are allocating `avifReformatState` on the stack. The two arrays will take 512KB of stack space, which can be a bit too big.

Reply:
I agree that this is much too large for something that is going to be on the stack, and not useful to AV1 conversion.